Automattic/woocommerce-payments
# [Simplifying Deposits UI] Adds the recent deposits table

Fixes #5700

#### Changes proposed in this Pull Request

Add a "Deposit History" section showing the three most recent deposits to the new Deposits Overview component.  The top of the Deposits Overview component is shown dimmed in the screenshot below as this has already been implemented in #5799 and is not part of this pull-request.  

![Screenshot 2023-03-24 at 2 25 31 PM copy](https://user-images.githubusercontent.com/57298/227401435-60ed61f3-399a-4aad-8a69-0cdf57d09448.png)

#### Screenshots

When the Deposits Overview component is loading, recent deposits are not shown in a loading state. A Customer may not have any recent deposits so we either need to show a loading state that then disappears or show no loading state and have content appear.

**Loading Screen**

![Screenshot 2023-03-24 at 2 45 28 PM](https://user-images.githubusercontent.com/57298/227404333-df03b42b-871c-460a-ade3-ab884cb5f06b.png)

**Partial Load**
![Screenshot 2023-03-24 at 2 45 31 PM](https://user-images.githubusercontent.com/57298/227404338-a0f22f4c-f0bb-4caf-b753-dcef5afe88a5.png)

**Fully Loaded**
![Screenshot 2023-03-24 at 2 45 34 PM](https://user-images.githubusercontent.com/57298/227404336-ba66434e-b6cb-4722-8e48-5224f8056474.png)


#### Testing instructions

1. Pull the changes and run `npm run build:deps`
2. Set `_wcpay_feature_simplify_deposits_ui` to `1` in the `wp-options` table
3. From WP Admin go to Payments to view the Overview page
4. Check the Next deposits tables loads as per the designs



-------------------


**Post merge**
